From 2fb0ba62af7583e96318bef92123054689399208 Mon Sep 17 00:00:00 2001
From: Christian Marangi <ansuelsmth@gmail.com>
Date: Tue, 14 Oct 2025 20:05:47 +0200
Subject: [PATCH 53/59] configs: enable PCS and EN8811 PHY for Airoha defconfig

Enable PCS and EN8811 PHY  for Airoha defconfig as they are present in
most of the RFB board.

Signed-off-by: Christian Marangi <ansuelsmth@gmail.com>
---
 configs/an7581_evb_defconfig | 1 +
 configs/an7583_evb_defconfig | 1 +
 2 files changed, 2 insertions(+)

diff --git a/configs/an7581_evb_defconfig b/configs/an7581_evb_defconfig
index d95aea44aef..4d69ebee0aa 100644
--- a/configs/an7581_evb_defconfig
+++ b/configs/an7581_evb_defconfig
@@ -72,6 +72,7 @@ CONFIG_SPI_FLASH_WINBOND=y
 CONFIG_SPI_FLASH_MTD=y
 CONFIG_AIROHA_ETH=y
 CONFIG_PCS_AIROHA_AN7581=y
+CONFIG_PHY_AIROHA_EN8811=y
 CONFIG_DM_MDIO=y
 CONFIG_CMD_MII=y
 CONFIG_CMD_MDIO=y
diff --git a/configs/an7583_evb_defconfig b/configs/an7583_evb_defconfig
index f9292f95207..fba494ca93e 100644
--- a/configs/an7583_evb_defconfig
+++ b/configs/an7583_evb_defconfig
@@ -71,6 +71,7 @@ CONFIG_SPI_FLASH_WINBOND=y
 CONFIG_SPI_FLASH_MTD=y
 CONFIG_AIROHA_ETH=y
 CONFIG_PCS_AIROHA_AN7583=y
+CONFIG_PHY_AIROHA_EN8811=y
 CONFIG_DM_MDIO=y
 CONFIG_CMD_MII=y
 CONFIG_CMD_MDIO=y
-- 
2.51.0

